### ISSUE TITLE
Can't choose options when creating PKI role in UI

### ISSUE THREAD

**Issue by db-adrian** [hashicorp/vault#6544]
**Describe the bug**
When I try to crate a PKI role in the UI all of the options are not filled out. I can create roles via the cli no problem, but the UI somehow does not show the inputs. 
**To Reproduce**
Follow the steps shown in this  [Tutotial](https://learn.hashicorp.com/vault/security/sm-pki-engine#web-ui-2)

**Expected behavior**

**Environment:**
* Vault Server Version (retrieve with `vault status`): 1.1.0
* Vault CLI Version (retrieve with `vault version`): v1.1.0
* Server Operating System/Architecture: Ubuntu 18.04.2 LTS

Vault server configuration file(s):

```hcl
listener "tcp" {
  address       = "0.0.0.0:8200"
  tls_cert_file = "/etc/vault.d/ssl/vault-server-bundle.pem"
  tls_key_file  = "/etc/vault.d/ssl/vault-server-key.pem"
}

api_addr = "https://****:8200" #dns name

storage "gcs" {
  bucket     = "***"
  ha_enabled = "true"
}

seal "gcpckms" {
  project     = "***"
  region      = "***"
  key_ring    = "***"
  crypto_key  = "***"
}

ui = true

```

**Additional context**
![screencapture-vault](https://user-images.githubusercontent.com/43702457/55684744-3b148600-594f-11e9-8708-27572246e6c8.png)

**Comment by evilhamsterman** [comment]
Same here in 1.4.4, running through the learn.hashicorp.com tutorials and ran into this. I can set the values using the API or the CLI but can't set or read them through the UI

**Comment by meirish** [comment]
Thanks for the report!

This was fixed by https://github.com/hashicorp/vault/pull/6474 which was included in 1.1.1 that was released yesterday.
